default-plugins/status-bar/src/main.rs

```diff
@@ -41,6 +32,153 @@ impl Display for LinePart {
     }
 }
 
+#[derive(Clone, Copy)]
+pub struct ColoredElements {
+    // slected mode
+    pub selected_prefix_separator: Style,
+    pub selected_char_left_separator: Style,
+    pub selected_char_shortcut: Style,
+    pub selected_char_right_separator: Style,
+    pub selected_styled_text: Style,
+    pub selected_suffix_separator: Style,
+    // unselected mode
+    pub unselected_prefix_separator: Style,
+    pub unselected_char_left_separator: Style,
+    pub unselected_char_shortcut: Style,
+    pub unselected_char_right_separator: Style,
+    pub unselected_styled_text: Style,
+    pub unselected_suffix_separator: Style,
+    // disabled mode
+    pub disabled_prefix_separator: Style,
+    pub disabled_styled_text: Style,
+    pub disabled_suffix_separator: Style,
+    // selected single letter
+    pub selected_single_letter_prefix_separator: Style,
+    pub selected_single_letter_char_shortcut: Style,
+    pub selected_single_letter_suffix_separator: Style,
+    // unselected single letter
+    pub unselected_single_letter_prefix_separator: Style,
+    pub unselected_single_letter_char_shortcut: Style,
+    pub unselected_single_letter_suffix_separator: Style,
+    // superkey
+    pub superkey_prefix: Style,
+    pub superkey_suffix_separator: Style,
+}
+
+impl ColoredElements {
```

Comment:
I'd personally not bother with the `new()` function, since the style fields are already public. I'd directly construct it as:
```rust
ColoredElements {
    selected_prefix_separator: // Something...
    // ...
}
```

Instead of calling `new()` in a similar way :)

Reply:
Yeah, that's fair